packages/ui/components/counter/counter.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { useState } from "react";
+
+import { counter } from "./counter.css";
+
+export const Counter: React.FC = () => {
```

Comment:
@Kyujenius  이거 명시안하는거 어떤가요~?

```suggestion
export const Counter = () => {
```